src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -498,9 +498,6 @@
   <data name="WRN_InvalidMainSig" xml:space="preserve">
     <value>'{0}' has the wrong signature to be an entry point</value>
   </data>
-  <data name="WRN_InvalidMainSig_Title" xml:space="preserve">
```

Comment:
I expected that if this one were unused, then, `WRN_InvalidMainSig` would also be unused. Is that not the case?

Reply:
No.  WRN_InvalidMainSig is used.